src/Samples/DesktopServer/Program.cs

```diff
@@ -1,31 +1,32 @@
 ﻿using System;
 using System.Linq;
 using System.ServiceModel;
-using CoreWCF.Samples.StandardCommon;
 
 namespace DesktopServer
```

Comment:
DesktopServer could be renamed to NetFrameworkCoreWCFServer

Reply:
Is the term NetFx known and understood outside of Microsoft?